packages/ui/cypress/tests/network-switch.cy.ts

```diff
@@ -0,0 +1,53 @@
+import { accountDisplay } from '../support/page-objects/components/accountDisplay'
+import { settingsPageWatchAccountUrl } from '../fixtures/landingData'
+import { settingsPage } from '../support/page-objects/settingsPage'
+import { topMenuItems } from '../support/page-objects/topMenuItems'
+import { watchMultisigs } from '../fixtures/watchAccounts/watchMultisigs'
+import { encodeAddress } from '@polkadot/keyring'
```

Comment:
Let's import it from `@polkadot/util-crypto` as we do elsewhere, which is a direct dependancy, unlike keyring

```suggestion
import { encodeAddress } from '@polkadot/util-crypto'
```